deployer/schema.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"github.com/flynn/flynn/Godeps/_workspace/src/github.com/flynn/go-sql"
+	"github.com/flynn/flynn/pkg/postgres"
+)
+
+func migrateDB(db *sql.DB) error {
+	m := postgres.NewMigrations()
+	m.Add(1,
+		`CREATE TABLE que_jobs (
+    priority    smallint    NOT NULL DEFAULT 100,
+    run_at      timestamptz NOT NULL DEFAULT now(),
+    job_id      bigserial   NOT NULL,
+    job_class   text        NOT NULL,
+    args        json        NOT NULL DEFAULT '[]'::json,
+    error_count integer     NOT NULL DEFAULT 0,
+    last_error  text,
+    queue       text        NOT NULL DEFAULT '',
+
+    CONSTRAINT que_jobs_pkey PRIMARY KEY (queue, priority, run_at, job_id))`,
+		`COMMENT ON TABLE que_jobs IS '3'`,
+	)
+	m.Add(2,
+		`CREATE EXTENSION IF NOT EXISTS "uuid-ossp"`,
+
+		`CREATE TYPE deployment_strategy AS ENUM ('all-at-once', 'one-by-one')`,
+		`CREATE TABLE deployments (
+    deployment_id uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+    app_id uuid NOT NULL,
+    old_release_id uuid NOT NULL,
+    new_release_id uuid NOT NULL,
+    strategy deployment_strategy NOT NULL,
+    status int8 NOT NULL DEFAULT 0,
+    created_at timestamptz NOT NULL DEFAULT now())`,
+
+		`CREATE SEQUENCE deployment_event_ids`,
+		`CREATE TABLE deployment_events (
+    event_id bigint PRIMARY KEY DEFAULT nextval('deployment_event_ids'),
+    deployment_id uuid NOT NULL REFERENCES deployments (deployment_id),
+    release_id uuid NOT NULL,
```

Comment:
Wouldn't `NULL` be better? I'm not a huge fan of signalling state via sentinel fields that are used for other things, is there a better way to do it? Columns are cheap.